docs/components/FormatTime/Web.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { CivilTime } from "@std-proposal/temporal";
+import { FormatTime } from "@jobber/components/FormatTime";
+
+export default {
+  title: "Components/Utilities/FormatTime/Web",
+  component: FormatTime,
+  parameters: {
+    viewMode: "story",
+    previewTabs: {
+      code: {
+        hidden: false,
+        extraImports: {
+          "@std-proposal/temporal": ["CivilTime"],
+        },
+      },
+    },
+  },
+} as ComponentMeta<typeof FormatTime>;
+
+const BasicTemplate: ComponentStory<typeof FormatTime> = args => (
+  <>
+    <FormatTime {...args} />
```

Comment:
I think the issues is with the way that we are passing in the args here 🤔

Reply:
Exactly, it doesn't handle well dates and react component passed through the args object. Thanks for catching it!